auto-update-dataset/ignore.csv

```diff
@@ -0,0 +1,2 @@
+﻿name
+tk.mybatis.mapper.annotation.IdTest.testCompositeKeys
```

Comment:
Should the auto update tool actually ignore this test or were you just using this test for testing purposes?

Were there some tests that we wanted the auto update tool to ignore that led you to add this feature?